src/ParentNode.php

```diff
@@ -37,4 +52,33 @@ private function prop_get_childElementCount() {
 	return $this->children->length;
 }
 
-}#
\ No newline at end of file
+/**
+ * @param string $selectors CSS selector(s)
+ * @param string $prefix
+ *
+ * @return HTMLCollection
+ */
+public function css(
+    string $selectors,
+    string $prefix = "descendant-or-self::"
+): HTMLCollection {
+    $converter = new CssSelectorConverter();
```

Comment:
There's room here to refactor CssSelectorConverter to a `Gt\Dom\CssSelectorConverter`, so it is easy to switch out with a non-symfony version, if the upcoming speed tests conclude that Symfony is too slow.